elixir/apps/domain/test/domain/actors_test.exs

```diff
@@ -3071,16 +2973,17 @@ defmodule Domain.ActorsTest do
       assert is_nil(other_actor.deleted_at)
     end
 
-    test "updates dynamic groups memberships", %{account: account, actor: actor, subject: subject} do
-      Fixtures.Actors.create_actor(type: :account_admin_user, account: account)
+    test "updates managed group memberships", %{account: account, actor: actor, subject: subject} do
+      new_actor = Fixtures.Actors.create_actor(type: :account_admin_user, account: account)
 
       group = Fixtures.Actors.create_managed_group(account: account)
 
       assert {:ok, actor} = delete_actor(actor, subject)
       assert actor.deleted_at
 
       group = Repo.preload(group, :memberships, force: true)
-      assert group.memberships == []
+      assert length(group.memberships) == 1
```

Comment:
This test had a bug. The number of actors in this test is `2`, so deleting one should result in a membership count for the managed group of `1`.